core/src/klio_core/config/core.py

```diff
@@ -177,9 +177,11 @@ class KlioJobConfig(object):
     version = utils.field(type=int)
 
     # optional attributes
+
     allow_non_klio_messages = utils.field(type=bool, default=False)
     metrics = utils.field(default={})
     blocking = utils.field(type=bool, default=False)
+    wait_for_pipeline_running = utils.field(type=bool, default=False)
```

Comment:
Mind adding just a little blurb [here in our config docs](https://github.com/spotify/klio/blob/develop/docs/src/userguide/config/job_config.rst) about this value? Just so that users know it's available to configure.

Reply:
👍🏾